### ISSUE TITLE
[Feature Request] Translation Removal in config menu

### ISSUE THREAD

**Issue by weearc** [plusls/MasaGadget#87]
### What new features do you want?

Currently translations have been added to masa's mod automatically when enable Masa-Gadget mod and the feature of disable translations can only be found at under `MiniHUD` config in Masa-Gadget's configuration menu. May confgs like this be added to `Tweakeroo` or `Litematica` ?
Config in `minihud`:
![image](https://github.com/plusls/MasaGadget/assets/19357165/b03dffb6-4a99-45f1-93c3-a9995a35ca2c)
Config in `tweakeroo`:
![image](https://github.com/plusls/MasaGadget/assets/19357165/ec81c1b1-cde3-4626-a93b-c774ec0fad11)

### Latest CI build tested

- [X] I confirm that the feature I requested is not in the latest CI build.

**Comment by Hendrix-Shen** [comment]
No, there is no plan at the moment, MasaGadget has built-in translation language files, just like Vanilla, the translation follows the language of the game itself, you can try to manually remove these files in the jar.
